modules/light-clients/08-wasm/go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/cosmos/ibc-go/modules/light-clients/08-wasm
 go 1.21
 
 retract (
+	// explicitly break api for client contracts
```

Comment:
Maybe something more explicit? `rename field of JSON-encoded verify messages from path to merlke_path`, or something like that.

Reply:
I updated with a more informative godoc message. Let me know what you think